src/Countries/England.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+
+class England extends Country
+{
+  public function countryCode(): string
+  {
+    return 'en';
+  }
+
+  protected function allHolidays(int $year): array
+  {
+    return array_merge([
+      'New Year\'s Day' => new CarbonImmutable($year . "-01-01", 'Europe/London'),
+      'Early May bank holiday' => new CarbonImmutable("first monday of may {$year}", 'Europe/London'),
```

Comment:
Could you use the format used in the other Country classes? Also, this seems to be a variable holiday. Makes more sense to put in the dedicated function? Same for Spring bank & Summer bank, I think